tests/blackbox/testlib/utils.py

```diff
@@ -95,18 +95,17 @@ def exec_command(cmd):
     :type cmd: list of str
     :returns: standard output
     :rtype: str
-    :raises AssertionError: if exit code is non-zero
+    :raises RuntimeError: if exit code is non-zero
     """
     exit_code, stdout_text, stderr_text = exec_test_command(cmd)
 
     expected_exit_code = 0
 
     if expected_exit_code != exit_code:
-        print("cmd = %s [%d != %d]" % (str(cmd), expected_exit_code, exit_code))
-        print("STDOUT= %s" % stdout_text)
-        print("STDERR= %s" % stderr_text)
-
-    assert expected_exit_code == exit_code
+        raise RuntimeError(
```

Comment:
You can get rid of ```expected_exit_code``` entirely now,  since it's only used in one place.

Reply:
Resolved.